EIPS/eip-7378.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 7378
+title: Add time-weighted averaging to the base fee
+description: Using geometric weights to average past block sizes into consideration
+author: Guy Goren (@guy-goren) <guy.nahsholim@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/add-time-weighted-averaging-to-the-base-fee-mechanism/15142
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-22
+---
+
+## Abstract
+
+The current formula for setting the base fee follows the original [EIP-1559](./eip-1559.md):
+
+$$b[i+1]\triangleq  b[i] \cdot \left( 1+\frac{1}{8} \cdot \frac{s[i]-s^* }{s^* }\right)$$
+
+where $i$ enumerates the blocks, $s^*$ is the desired block size, $b[i]$ is the base fee at block $i$, and $s[i]$ is the size of the block at height $i$. This formula considers only the last block size $s[i]$. This mechanism incentivizes proposers to collude with users to manipulate the base fee.
```

Comment:
```suggestion
only considers the last block size $s[i]$. This mechanism incentivizes proposers to collude with users to manipulate the base fee.
```
The abtract should be terse. Less-important details like this can and should be omitted.